harrystech/target-s3-jsonl
# Init target-s3-jsonl

## Description

Implement target-s3-jsonl

## QA

You can run:
```
meltano elt tap-carbon-intensity target-s3-jsonl
```

This tap has also been tested in dev:
see https://github.com/harrystech/harrys-data-warehouse/pull/2453
